lib/plugin.py

```diff
@@ -70,18 +95,41 @@ def create_push_details(self):
         return {k: v for k, v in push_details if v is not None}
 
     def _provision_user(self):
-        phone_attribute = self.plugin_configuration.get('starling_auto_provision', 'phone_attribute')
-        email_attribute = self.plugin_configuration.get('starling_auto_provision', 'email_attribute')
-        name_attribute = 'displayName'
-        if phone_attribute and email_attribute:
+        cached_user_info = self.__cache.get(USER_ID_CACHE_KEY_PREFIX)
```

Comment:
get must use the constructed _user_id_cache_key
the whole logic here needs to be inside the "if" form line 108
if self._ldap_user_info.phone and self._ldap_user_info.email:

Reply:
Done